### DISCUSSION TITLE
[Bug] using Module Federation with an additional Entrypoint prevents availability of init and get

### DISCUSSION THREAD

**Discussion by MariusZeger** [webpack/webpack#17177] [Q&A]
# Bug report

**What is the current behavior?**

i am using module federation with two angular 15 projects.

now i wanted to add an additional entrypoint script ( like described in your [docs ](https://webpack.js.org/concepts/module-federation/#infer-publicpath-from-script)), but doing this creates the issue that the init and get functions are not available anymore

![image](https://github.com/webpack/webpack/assets/40056245/f4910bf0-c825-45ea-a074-bbd845b1dc5b)

**If the current behavior is a bug, please provide the steps to reproduce.**

using a webpack config like this works:

`module.exports = {
    output: {
        publicPath: "http://localhost:4201/",
        uniqueName: "appAnalyzer",
        scriptType: "text/javascript"
    },
    optimization: {
        runtimeChunk: false
    },
    plugins: [
        new ModuleFederationPlugin({
            name: "appAnalyzer",
            filename: "remoteEntry.js",
            exposes: {
                ...
            },
        })
    ]
};`

using a webpack.config with an additional entrypoint creates the issue:

`module.exports = {
    entry: {
        main: "./app/main.ts"
        appAnalyzer: "./app/setPublicPath.js" // same with  "./app/main.ts"
    },
    output: {
        publicPath: "http://localhost:4201/",
        uniqueName: "appAnalyzer",
        scriptType: "text/javascript"
    },
    optimization: {
        runtimeChunk: false
    },
    plugins: [
        new ModuleFederationPlugin({
            name: "appAnalyzer",
            filename: "remoteEntry.js",
            exposes: {
                ...
            },
        })
    ]
};`

**What is the expected behavior?**

the get and init Methods should still be available 

i debugged the generated "remoteEntry.js" files and noticed that here is a difference 

![image](https://github.com/webpack/webpack/assets/40056245/98249055-eacb-4298-9694-c4d7819804d6)

i created a small sciprt in my frame appliction and
switching 12196 with 34731 in the loaded files resolves the issue.
(with 34731being the entrypoint script and 12196 the container entry script)

**Other relevant information:**
webpack version:  5.81
Node.js version: 16.16.0

my package.json contains

'
"webpack": "5.81.x",
"@angular/common": "15.2.8",
"@angular/core": "15.2.8",
"@angular/cdk": "15.2.8",
"@angular/cli": "15.2.7",
'

Thanks in Advance

**Comment by alexander-akait** [comment]
Please open an issue in `angular-cli` repo, thank you
